charts/media-api-service/readme.md

```diff
@@ -0,0 +1,42 @@
+# Microfront end helm chart
+
+A sample library for dealing with microfront ends that deploy behind traefik.
+
+## Installation
+
+clone the [helm-charts](https://github.com/htaic/helm-charts) repo into your source code and move the example file out:
+
+```bash
+git clone https://github.com/htaic/helm-charts.git
+mv ./helm-charts/charts/media-token-service/values.yaml ./DevValues.yaml
```

Comment:
you should update the readme or remove it

Reply:
removed